apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/dashboard/_components/BatchEvaluationsTable.tsx

```diff
@@ -45,7 +45,7 @@ export default function BatchEvaluationsTable({
                   .detail({ id: project.id })
                   .commits.detail({ uuid: commit.uuid })
                   .documents.detail({ uuid: document.documentUuid })
-                  .evaluations.detail(evaluation.uuid).root,
+                  .evaluations.detail(evaluation.id).root,
```

Comment:
didn't we agree on using uuid?

Reply:
What connect service produce doesn't have `uuid`. It has `id`. 
https://github.com/latitude-dev/latitude-llm/blob/main/packages/core/src/services/evaluations/connect.ts#L97

It was changing this service or changing the route. I prefer changing the route